console-subscriber/src/lib.rs

```diff
@@ -201,27 +204,12 @@ impl TasksLayer {
                 // this shouldn't happen, since we trigger a flush when
                 // approaching the high water line...but if the executor wait
                 // time is very high, maybe the aggregator task hasn't been
-                // polled yet. so, try to handle it gracefully...
-                tracing::warn!(
-                    "console buffer is full; some task stats may be delayed. \
-                     preemptive flush interval should be adjusted..."
-                );
-                let tx = self.tx.clone();
-                tokio::spawn(async move {
-                    if tx.send(event).await.is_err() {
-                        tracing::debug!("task event channel closed after lag");
-                    }
-                });
```

Comment:
Since we are now dropping events here, we should probably also ensure that the aggregator task will gracefully handle cases where it sees an event for a task with an ID it hasn't seen before. I think right now, it may record stats that are inaccurate, because it uses `update_or_default` on `TaskData` in a few places. Instead, we should probably ensure that if, for example, we never observed the creation of a task ID, we don't record any future timestamps for it (since they'll be wrong).

Reply:
also, we now don't really have a way to record cases where we missed events, which is probably worth doing (since it could indicate a console configuration error). 

perhaps we might want to have an `AtomicUsize` or something here that's incremented whenever we drop an event, and then in the aggregator task, we could log a warning when that counter is > 0? this wouldn't be a problem since the logging would occur in the aggregator task, outside of the subscriber. but, we can do that in a follow-up change!